src/contexts/accessControlContext/index.jsx

```diff
@@ -24,7 +24,7 @@ import T from "prop-types";
 
 const defaultValues = {
   accessControl: null,
-  accessControlType: null,
+  accessControlType: "",
```

Comment:
this one should be null, since the value is set from the return value of the `useAccessControlType` hook, and we check whether that value is falsy in the `useAccessControl` hook.

Reply:
It doesn't necessarily mean that there is not an access control type, it just means that for whatever reason we couldn't determine it. The SDK methods used return booleans, and we check for an ACR or an ACL.

Reply:
In addition, f we're updating the default values here we will need to update it in the corresponding hook and related code and docs as well.

Reply:
> was successful but the answer is that the user doesn't have access control instead of a null kind of leaves me guessing, was the call ever made?

We do check for the existence of either an error or a response to see if the hook is still validating so that could potentially answer that question.